package.json

```diff
@@ -32,13 +32,13 @@
     "common-js-file-extensions": "1.0.2",
     "compression": "1.7.1",
     "connect": "3.6.5",
-    "cookie": "0.3.1",
-    "cookie-parser": "1.4.3",
-    "cookie-signature": "1.0.6",
-    "csurf": "1.9.0",
+    "cookie": "0.4.0",
+    "cookie-parser": "1.4.4",
+    "cookie-signature": "1.1.0",
+    "csurf": "1.10.0",
     "ejs": "2.5.7",
-    "express": "4.16.2",
-    "express-session": "1.15.6",
+    "express": "4.17.1",
+    "express-session": "git+https://github.com/expressjs/session.git",
```

Comment:
Waiting for express-session 1.16.3 or higher

Reply:
@pixnlove [express-session 1.17.0](https://www.npmjs.com/package/express-session/v/1.17.0) is out 👍

Reply:
Done @johnabrams7, waiting for review @rachaelshaw